app/assets/javascripts/registration/grid.js.erb

```diff
@@ -470,6 +470,17 @@ var gridContext = function() {
           }
         })
 
+        // Show/hide external workflow drop-down when workflow selector changes
```

Comment:
am i missing other integration tests for the reg form? (other than it shows without an error)

is there an existing pattern for adding what will be a pretty simple test like this in?